docker-compose.yml

```diff
@@ -1,4 +1,15 @@
 services:
+  db:
+    image: postgres
```

Comment:
please indicate the version / tag of the image we are using

Reply:
16